src/app/accounts/lock.component.ts

```diff
@@ -38,20 +34,19 @@ export class LockComponent extends BaseLockComponent implements OnDestroy {
 
     constructor(router: Router, i18nService: I18nService,
         platformUtilsService: PlatformUtilsService, messagingService: MessagingService,
-        userService: UserService, cryptoService: CryptoService,
-        storageService: StorageService, vaultTimeoutService: VaultTimeoutService,
+        cryptoService: CryptoService, vaultTimeoutService: VaultTimeoutService,
         environmentService: EnvironmentService, stateService: StateService,
         apiService: ApiService, private route: ActivatedRoute,
         private broadcasterService: BroadcasterService, ngZone: NgZone,
         logService: LogService, keyConnectorService: KeyConnectorService) {
-        super(router, i18nService, platformUtilsService, messagingService, userService, cryptoService,
-            storageService, vaultTimeoutService, environmentService, stateService, apiService, logService,
+        super(router, i18nService, platformUtilsService, messagingService, cryptoService,
+            vaultTimeoutService, environmentService, stateService, apiService, logService,
             keyConnectorService, ngZone);
     }
 
     async ngOnInit() {
         await super.ngOnInit();
-        const autoPromptBiometric = !await this.storageService.get<boolean>(ConstantsService.disableAutoBiometricsPromptKey);
+        const autoPromptBiometric = !await this.stateService.getNoAutoPromptBiometrics();
```

Comment:
A lot of the names aren't great. I didn't want to mess with them because of the increased risk, but dealing with them later will be somewhat difficult and likely require more state migrations. Certainly something to think about. I'm going to log a [tech debt item](https://app.asana.com/0/1201461753012676/1201520488476169).